clang/lib/StaticAnalyzer/Core/LoopUnrolling.cpp

```diff
@@ -226,6 +226,21 @@ static bool isPossiblyEscaped(ExplodedNode *N, const DeclRefExpr *DR) {
           return false;
       }
     }
+
+    if (const SwitchStmt *SS = dyn_cast<SwitchStmt>(S)) {
+      if (const CompoundStmt *CST = dyn_cast<CompoundStmt>(SS->getBody())) {
```

Comment:
```suggestion
    if (const auto *SS = dyn_cast<SwitchStmt>(S)) {
      if (const auto *CST = dyn_cast<CompoundStmt>(SS->getBody())) {
```

The type is already mentioned on the line.